src/firebase/notifications.ts

```diff
@@ -59,28 +58,28 @@ function* handlePaymentReceived(
   notificationState: NotificationReceiveState
 ) {
   if (notificationState !== NotificationReceiveState.AppAlreadyOpen) {
-    const info: RecipientInfo = yield select(recipientInfoSelector)
     const address = transferNotification.sender.toLowerCase()
 
-    navigateToPaymentTransferReview(
-      TokenTransactionType.Received,
-      new BigNumber(transferNotification.timestamp).toNumber(),
-      {
+    yield call(navigate, Screens.TransactionDetailsScreen, {
+      transaction: {
+        __typename: 'TokenTransferV2',
+        type: TokenTransactionTypeV2.Received,
+        transactionHash: transferNotification.txHash,
+        timestamp: new BigNumber(transferNotification.timestamp).toNumber(),
+        block: transferNotification.blockNumber,
+        address,
         amount: {
-          value: divideByWei(transferNotification.value),
-          currencyCode: mapOldCurrencyToNew(transferNotification.currency),
+          value: transferNotification.value,
+          tokenAddress: transferNotification.tokenAddress,
         },
-        address,
-        comment: transferNotification.comment,
-        recipient: getRecipientFromAddress(
-          address,
-          info,
-          transferNotification.name,
-          transferNotification.imageUrl
-        ),
-        type: TokenTransactionType.Received,
-      }
-    )
+        metadata: {
+          title: transferNotification.name,
+          image: transferNotification.imageUrl,
+          comment: transferNotification.comment,
+        },
+        fees: [], // TODO?
```

Comment:
i think it's fair enough to omit fees 😅 it'd be strange to see that fees were deducted from a payment